ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -364,12 +359,22 @@ class H2OGridSearch(override val uid: String) extends Estimator[H2OMOJOModel]
   }
 
   override def copy(extra: ParamMap): this.type = defaultCopy(extra)
+
+  private def extractH2OParameters(algo: H2OSupervisedAlgorithm[_, _, _ <: Model.Parameters]): Model.Parameters = {
+    val m = algo.getClass.getDeclaredMethod("updateH2OParams")
+    m.setAccessible(true)
+    m.invoke(algo)
+
+    val field = PojoUtils.getFieldEvenInherited(algo, "parameters")
+    field.setAccessible(true)
+    field.get(algo).asInstanceOf[Model.Parameters]
+  }
 }
 
 object H2OGridSearch extends H2OParamsReadable[H2OGridSearch] {
 
   object SupportedAlgos extends Enumeration {
-    val gbm, glm, deeplearning, xgboost = Value // still missing pipeline wrappers for KMeans & drf
+    val gbm, glm, deeplearning, xgboost = Value
```

Comment:
Couldn't we add DRF as well?

Reply:
Exactly!, but that will go only to master as DRF is in master only - see this jira https://0xdata.atlassian.net/browse/SW-1588